planemo/bioconda_scripts/bioconductor_skeleton.py

```diff
@@ -0,0 +1,482 @@
+#!/usr/bin/env python
+
```

Comment:
I didn't make the decision and I don't have a strong opinion - but it would be helpful then if you'd put these scripts in pypi right? Planemo from pypi should be able to build, lint, etc... conda packages.